client/src/pages/Therapist/MyAccount/index.js

```diff
@@ -181,7 +181,7 @@ const MyAccount = () => {
       return getMediaUrl(uploadedFileInfo);
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [uploadedFileInfo?.id]);
+  }, [uploadedFileInfo]);
```

Comment:
there is no id in the `uploadedFileInfo` but this shouldn't be an object also you may add something unique in the object like ` [uploadedFileInfo?.key, uploadedFileInfo?.uploadedToS3]`
```suggestion
  },  [uploadedFileInfo?.key, uploadedFileInfo?.uploadedToS3]);
```

Reply:
Hey @fadeomar this is what me and @RamyAlshurafa were discussing and it's agreed we can have this as an object for now as otherwise it doesn't work